.github/workflows/job-postcommit-java-tpcds-dataflow.yml

```
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test for PostCOmmit Java TPCDS using dataflow runner.
+name: PostCommit Java Tpcds Dataflow
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+permissions: read-all
+
+jobs:
+  java-postcommit-tpcds-dataflow:
+    name: Run PostCommit Java Tpcds Dataflow 
+    runs-on: [self-hosted, ubuntu-20.04]
+    strategy:
+      fail-fast: false
+    timeout-minutes: 240
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false          
+      - name: Setup environment
+        uses: ./.github/actions/setup-self-hosted-action 
+      - name: Run *** RUN TPCDS IN BATCH MODE USING DATAFLOW RUNNER ***
```

```suggestion
      - name: Run TPCDS in Batch mode on Dataflow
```

Same comment applies to the similarly named items below